src/main/java/com/api/parkingmeter/application/usecase/UpdateParkingSession.java

```diff
@@ -6,19 +6,27 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
+import java.math.BigDecimal;
+
 @Component
 @RequiredArgsConstructor
 public class UpdateParkingSession {
   private final ParkingSessionGateway parkingSessionGateway;
 
   public ParkingSession execute(final String licensePlate) {
-    final var parkingSession =
+    ParkingSession parkingSession =
```

Comment:
Não tinha a necessidade de mudar, mas já que vc quis, a troca é o var pelo ParkingSession, mas precisa devolver o final, ficando:
```suggestion
final ParkingSession parkingSession =
```